src/check_update.c

```diff
@@ -63,13 +65,24 @@ enum swupd_code check_update()
 		ret = SWUPD_SERVER_CONNECTION_ERROR;
 	}
 
+	if (globals.verbose) {
```

Comment:
Check for log level here so it will work with --debug too

    if (log_get_level() >= LOG_INFO_VERBOSE)

Reply:
Good catch! Didnt think about that. Thanks!

Reply:
Done